framework/application/CMakeLists.txt

```diff
@@ -4,9 +4,18 @@ target_sources(gfxrecon_application
                PRIVATE
                     application.h
                     application.cpp
-                    $<$<BOOL:${XCB_FOUND}>:xcb_application.h xcb_window.h xcb_application.cpp xcb_window.cpp>
-                    $<$<BOOL:${WAYLAND_FOUND}>:wayland_application.h wayland_window.h wayland_application.cpp wayland_window.cpp>
-                    $<$<BOOL:${WIN32}>:win32_application.h win32_window.h win32_application.cpp win32_window.cpp>
+                    $<$<BOOL:${XCB_FOUND}>:xcb_application.h>
```

Comment:
Mostly just curiosity: is there a reason these have been reformatted to be one per line?